AbletonMCP_Remote_Script/__init__.py

```
@@ -939,6 +994,342 @@ def _get_scene_info(self, scene_index):
+            # Set the name if provided
+            if new_cue_point and name:
+                new_cue_point.name = name
+
+            # Restore original song time
+            self._song.current_song_time = original_time
+
+            return {
+                "created": True,
+                "time": target_time,
+                "name": name if name else (new_cue_point.name if new_cue_point else "")
+            }
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)